docs/architecture/components/1_batcher.md

```diff
@@ -0,0 +1,13 @@
+# Batcher
+
+Batcher is the Entity in charge of recieving User's proofs.
+
+The Batcher will recieve proofs from different Users, build a merkle root from these proofs, upload the proofs to an S3 (so that the operators can download them), and submit this information to the [Aligned Service Manager](./3_service_manager_contract.md).
+
+To avoid trust assumptions, the Batcher has some specific interesting mechanisms;
+
+So that the User is 100% sure that his proof was included in a batch, the Batcher will send to each User the Merkle Proof (or Merkle Path). With this, the User can rebuild the Merkle Root starting from his proof, thus verifying his proof was actually included in the batch.
+
+Also, to avoid unnecesarry proof submitions, the Batcher does a preliminary verification of the submitted proofs, to avoid as much as possible the submition of false proofs in a batch.
```

Comment:
```suggestion
Also, to avoid unnecesarry proof submissions, the Batcher does a preliminary verification of the submitted proofs, to avoid as much as possible the submission of false proofs in a batch.
```

Reply:
Also, I would replace the second "submission" with something like "presence" to avoid repeating so much. "to avoid as much as possible the presence of false proofs in a batch".